src/Components/Article.jsx

```diff
@@ -51,21 +53,35 @@ const Article = ({username}) => {
         })
     }
 
+    const handleVote = (num) => {
+        const newVote = votes + num
+        setVotes(newVote)
+        updateVotesById(article_id, num).catch((err) => {
+            setVotes(votes)
```

Comment:
We don't need to do this again as it will just be setting it to be the same

Reply:
Just need to patch it - already updated the patch state, but we should have a catch block to remove -1 from the votes if the patch fails